spec/controllers/admin/csv_export_spec.rb

```diff
@@ -25,12 +25,14 @@
 
       expect(user["first_name"]).to eq(@person.given_name)
       expect(user["last_name"]).to eq(@person.family_name)
+      expect(user["display_name"]).to eq(@person.display_name || "")
```

Comment:
I'm not completely sure why you need to use OR `||` here. Is there some unknown configuration coming outside which affects to the output, or why it's not known what will be the end result?